swift/ql/lib/codeql/swift/dataflow/ExternalFlow.qll

```diff
@@ -498,7 +498,7 @@ predicate parseContent(AccessPathToken component, Content content) {
   parseEnum(component, content)
   or
   component.getName() = "ArrayElement" and
```

Comment:
I think this could be surprising to a future reader.  Perhaps we should comment it?
```suggestion
  // map legacy "ArrayElement" specification components to `CollectionContent`
  component.getName() = "ArrayElement" and
```